packages-web/calendar/tests/e2e/specs/onCreate.spec.ts

```diff
@@ -0,0 +1,29 @@
+import page from "../../../../../configs/e2e/src/pages/page";
+import Calendar from "../objects/calendar.widget";
+
+describe("Calendar", () => {
+    beforeEach(() => {
+        page.open("p/onCreate");
+    });
+
+    it("should open create event page with Microflow", () => {
+        const calendar = new Calendar("calendar1");
+
+        calendar.element.waitForExist();
+        calendar.calendarSlot.click();
+        page.modalDialog.waitForExist();
+
+        expect(page.modalDialogHeader.getText()).toContain("Edit Event (Microflow)");
```

Comment:
Is it about editing an event or creating one?

Reply:
I think logic is indeed the same, with edit/new - edit page opened